shipyard-controller/handler/debughandler.go

```diff
@@ -0,0 +1,221 @@
+package handler
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"sort"
+
+	"github.com/gin-gonic/gin"
+	apimodels "github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/keptn/shipyard-controller/common"
+	"github.com/keptn/keptn/shipyard-controller/models"
+)
+
+type IDebugHandler interface {
+	GetAllProjects(context *gin.Context)
+	GetSequenceByID(context *gin.Context)
+	GetAllSequencesForProject(context *gin.Context)
+	GetAllEvents(context *gin.Context)
+	GetEventByID(context *gin.Context)
+}
+
+type DebugHandler struct {
+	DebugManager IDebugManager
+}
+
+func NewDebugHandler(debugManager IDebugManager) *DebugHandler {
+	return &DebugHandler{
+		DebugManager: debugManager,
+	}
+}
+
+// GetAllProjects godoc
+// @Summary      Get all keptn projects
+// @Description  Get all keptn projects
+// @Tags         Project
+// @Success      200                  {object}  []apimodels.ExpandedProject     "ok"
+// @Failure      400                  {object}  models.Error                    "Bad Request"
+// @Failure      500                  {object}  models.Error                    "Internal error"
+// @Router       /debug/project [get]
+func (dh *DebugHandler) GetAllProjects(c *gin.Context) {
```

Comment:
do we actually need this endpoint? this seems to be a copy of https://github.com/keptn/keptn/blob/a66ac61809eaa8167f9f78f8fa4f200eaf344a3e/shipyard-controller/handler/projecthandler.go#L197